crow/faq.md

```diff
@@ -123,6 +123,33 @@ input[1].change = function(s) output[1](s) end
 input[1].mode = 'change'
 ```
 
+### Max
+
+#### what are the formatting differences between druid + Max syntax?
+
+In Max, you use the exact same command as you would in druid - the only difference is you wrap them in quotes to make sure they are a single symbol, and prepend that symbol with “tell_crow”
+
+In druid:
+`output[1].volts = 3`
+
+In Max:
+`tell_crow “output[1].volts = 3”`
+
+The quotes are crucial because they make sure the Lua command is treated a single chunk.
+
+#### best practices
+
+The `[sprintf]` object is the easiest way to format messages to crow. Using the `symout` argument, the `[sprintf]` object outputs a string as a single symbol -- ideal for sending to crow as a Lua chunk.
+
+*Example: control the voltage of an assignable output*
+
+![](images/max-faq-1.png)
+
+- `[sprintf symout "output[%i].volts = %.2f"]` creates a string that accepts an integer (output ID) through its first inlet and a 2-decimal float (voltage) through its second inlet.
```

Comment:
consider removing the trailing period after "inlet" (or adding one to the other list items to stay consistent)?